From ad5a23903c1ab938df8a02b4fc22ae26bbc9fcfb Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 23 Mar 2022 14:30:57 +0200
Subject: [PATCH] [SW-69581] gaudi3: adjust boot descriptors address

For gaudi3 boot descriptors are stored in the 6th sector of the
spi flash.
flash structure spreadsheet:
https://habanalabs.sharepoint.com/:x:/g/ERLTEjwRuoFHqr1mgqFoBIIBYaUKhLbV3bC9M72cQpZigw?e=2T8f5j

Change-Id: Ib7b3db94d222cb940088d7f6d9f829227f81555d
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index a4cb6d4..cb7de0b 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -11,7 +11,7 @@
 #define PPBOOT_SIZE			0x4000		/* 16KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 
-#define HL_FW_BOOT_PTRS_BASE		0x80000
+#define HL_FW_BOOT_PTRS_BASE		0x60000
 
 #define HL_SP_SRAM_BASE			0x300007FD3120000ULL
 #define HL_SP_SRAM_SIZE			SZ_64K
